OCCtask1/src/main/java/com/example/OCCtask1/ProjectController.java

```diff
@@ -0,0 +1,63 @@
+package com.example.OCCtask1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Optional;
+
+
+@RestController
+public class ProjectController {
+
+    @Autowired
+    private ProjectRepository projectRepository;
+
+    @GetMapping(value = "/api/projects")
+    public List<Project> getAllProjects() {
+        System.out.println("Getting all projects.");
+        return projectRepository.findAll();
+    }
+
+    @GetMapping(value = "/api/projects/{projectId}")
+    public Project getProjectById(@PathVariable String projectId){
+        System.out.println("Getting project with ID: {}" + projectId);
+        checkId(projectId);
+        Optional OptionalProject = projectRepository.findById(projectId);
+        if (!OptionalProject.isPresent()) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "not find document", new FileNotFoundException());
```

Comment:
```suggestion
            throw new ResponseStatusException(HttpStatus.NOT_FOUND, String.format("Not found project with id=%s", projectId));
```
1. 404 response status is better
2. The exception message should be more detailed
3. The definition of 'cause' is not needed and `FileNotFoundException` is not applicable in this case